mc/include/alps/mc/mpiadapter.hpp

```diff
@@ -72,15 +72,15 @@ namespace alps {
                             alps::mpi::all_reduce(communicator,
                                                   has_count,
                                                   std::plus<size_t>());
-                    if (sum_counts == communicator.size()) {
+                    if ((int)sum_counts == communicator.size()) {
                         if (communicator.rank() == 0) {
                             typename Base::observable_collection_type::value_type merged = this->measurements[*it];
                             merged.collective_merge(communicator, 0);
                             partial_results.insert(*it, merged.result());
                         } else {
                             this->measurements[*it].collective_merge(communicator, 0);
                         }
-                    } else if (sum_counts > 0 && sum_counts < communicator.size()) {
+                    } else if (sum_counts > 0 && (int)sum_counts < communicator.size()) {
```

Comment:
The same: not a big deal, but why C-style cast `(int)` rather than `static_cast<int>()`, for consistency with other places?